README.md

```diff
@@ -75,7 +75,7 @@ By default, React will use the live server backend to allow seamless front end d
 
 ### Server Notes
 
-The staging and production servers are both hosted on `360-frontend.gordon.edu` (which runs Windows). This machine is also known as `360React.gordon.edu` (used by Travis CI), `360train.gordon.edu`, and `360.gordon.edu`.
+The staging and production servers are both hosted on `360-frontend.gordon.edu` (which runs Windows). This machine is also known as `360React.gordon.edu`, `360train.gordon.edu`, and `360.gordon.edu`.
```

Comment:
If the name 360React.gordon.edu was just used by Travis, could we remove that name from the documentation? Or do we know of another place it is used?

Reply:
It's not needed anywhere but technically none of these are needed. I figured it was better to leave them in so that people don't get confused thinking they're different servers.